docs/reference/ml/apis/jobcounts.asciidoc

```diff
@@ -65,8 +65,7 @@ or old results are deleted, the job counts are not reset.
   (long) The number of bucket results produced by the job.
 
 `earliest_record_timestamp`::
-  (string) The timestamp of the earliest chronologically ordered record.
-  The datetime string is in ISO 8601 format.
+  (date) The timestamp of the earliest chronologically processed record.
```

Comment:
Maybe change “processed record” to “input document”.

Same on line 98/99.

Reply:
Thanks @droberts195 I implemented those changes.